scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -407,8 +419,6 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
         vd = vd.annotate_entries(FT=~ft.any_no & (ft.any_yes | ((~ft.any_snp | ft.any_snp_ok) & (~ft.any_indel | ft.any_indel_ok))))
 
         vd = vd.drop('allele_NO', 'allele_YES', 'allele_is_snp', 'allele_OK')
+        vd = vd.rename({'as_vqsr': 'as_vets'})
```

Comment:
hmmmmm hopefully there is no 'as_vqsr' field in the VDS anymore? I assume if the field doesn't exist, the code doesn't fall over, but still feels like a one of vestige of hardcoding in Echo

Reply:
note also that we have things like     
 vd = vd.annotate_rows(as_vqsr=vd.as_vqsr.map_values(lambda value: value.drop('yng_status')))
that it would be lovely to have a follow on pr get rid of for the sake of sanity and cleanliness

Reply:
As discussed, this will be resolved in the ticket to remove VQSR from VDS creation